_content/setup-wallets.md

```diff
@@ -6,82 +6,60 @@ With Interledger, all currency pairs that senders and receivers use are supporte
+2. Because your new accounts don't not have any test funds, you will want to add value to them. In Xpring, click the blue `Make it rain` button to get free money! In Rafki, click the `Add Funds` button.
+3. If you want someone to pay you, provide them with your payment pointer. That's it!
 
-1. Open [rafiki.money](https://rafiki.money).
-2. Click **Create an account**.<br>
-   A **Create an Account** box appears.
-3. Enter the name and password you want for this account. Click **Signup**.<br>
-    The rafiki.money site provides you a payment pointer with your selected name. You must next create a default account for your payment.  
-4. Click **Add account**.<br>
-   The **Create account** window appears.
-5. Enter a name for your account and click **Create**.<br>
-   You next have to assign a default account for your payment pointer.        
-6. Click **No default account set for Payment pointer. Click here to assign default account**.
-7. From the **Default Account** dropdown list, select an account and click **Submit**.<br>
-   You can create additional accounts as desired. Each of these accounts belongs to the same payment pointer. You can send test XRP from any of these accounts, but the default account will be the recipient of any test XRP sent from another payment pointer.
-8. To add test XRP to an account, click to select the account on the home page. Click **Add Funds** to add 100 test XRP to your account. Repeat as desired.
+Note that when you subsequently login to either testnet account, you will see the current state of your wallet with the current amount of test XRP. Your accounts can receive test XRP when you are not logged in.   
 
-When you next visit rafiki.money, log in to work with the accounts you have set up.
+### 3. Send Money Between Interledger Wallets
+Once you have testnet funds in one of your wallets, you can send those funds to any other Interledger-enabled wallet.
 
-### Send money with your rafiki.money payment pointer
+1. Sign in to your account at [https://xpring.io](https://xpring.io)
+2. Look at the **Send** portion of the page on the right.
+3. Enter the number of test XRP that you want to send.
+4. Enter the payment pointer for the address you want to send test XRP to (hint: try one from [https://rafiki.money](https://rafiki.money)).
+5. Click **Send**.
 
-With rafiki.money, a payment pointer has a default account to receive test funds from another payment pointer. You can create additional financial accounts under the same payment pointer that can also store test funds and be used to send test funds. These examples continue to use test XRP.
+The amount of test XRP in the lower left adjusts to the new value. At the same time, you should see your balance in rafki.money go down.
```

Comment:
At the same time, if you have your rafiki.money wallet account open in another tab, you will see your balance in that wallet account go down.

Reply:
Fixed.